bundles/org.openhab.binding.tibber/src/main/java/org/openhab/binding/tibber/internal/handler/TibberHandler.java

```diff
@@ -552,19 +569,21 @@ public void onClose(int statusCode, String reason) {
         @OnWebSocketError
         public void onWebSocketError(Throwable e) {
             String message = e.getMessage();
-            logger.debug("Error during websocket communication: {}", message);
+            logger.error("Error during websocket communication: {}", message);
```

Comment:
```suggestion
            logger.warn("Error during websocket communication: {}", message);
```
Maybe better to leave to debug at all, as the mechnism you is already automatically reconnecting?!

Reply:
The level 'error' or 'warn' is ok.
I would not choose debug because with this method also some other errors during the communication could be shown. With 'error' or 'warn' we are able to support without debug logs.